### PR TITLE
Add composer

### PR DESCRIPTION
**Pull request by roxblnfk**
- merge this pull request
- add the version tag `3.10.1` to the resulting commit
- go to [packagist.org](https://packagist.org/packages/submit) and register it!
- close Issue #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+
+Installation
+------------
+
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+
+Either run
+
+```
+php composer.phar require skrol29/tinybutstrong "*"
```
Comment (simonschaufi) [inline README.md]: this should be just `composer require skrol29/tinybutstrong`

Comment (roxblnfk) [reply README.md]: fair remark

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+Installation
+------------
+
+The preferred way to install this extension is through [composer](http://getcomposer.org/download/).
+
+Either run
+
+```
+php composer require skrol29/tinybutstrong "*"
```
[the remark this answers is not in the document]
Comment (roxblnfk) [reply README.md]: The irony is that this change does not change the status of the pull request

[the remark this answers is not in the document]
Comment (simonschaufi) [reply README.md]: Yeah, I don't know why @Skrol29 doesn't respond.

**Comment by simonschaufi** [comment]
I wrote Skrol29 a direct email and hope to hear from him.

**Comment by simonschaufi** [comment]
Yeah, I'm glad you have access now to the new composer vendor name!

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "tinybutstrong/tinybutstrong",
+    "description": "Template Engine for Pro and Beginners ",
```
Comment (simonschaufi) [inline composer.json]: remove last space

**Comment by roxblnfk** [comment]
Thanks! Nice job!
Maybe next will be the [opentbs](https://github.com/Skrol29/opentbs/pull/1)?

**Pull request merged**